api/hasura/actions/_handlers/allocationCsv.ts

```diff
@@ -115,63 +117,61 @@ export function generateCsvValues(
   giftTokenSymbol = circleDist ? circleDist.vault.symbol : giftTokenSymbol;
   const { users } = circle;
 
-  return users.map((u, idx) => {
-    const received = u.received_gifts.length
-      ? u.received_gifts
-          .map(g => g.tokens)
-          .reduce((total, tokens) => tokens + total)
-      : 0;
-    const claimed = !fixedDist
-      ? 0
-      : fixedDist.claims
-          .filter(c => c.profile_id === u.profile?.id)
-          .reduce((t, g) => t + g.new_amount, 0) || 0;
-    let circle_claimed = !circleDist
-      ? 0
-      : circleDist.claims
-          .filter(c => c.profile_id === u.profile?.id)
-          .reduce((t, g) => t + g.new_amount, 0) || 0;
-
-    if (
-      circleDist &&
-      circleDist.distribution_type === DISTRIBUTION_TYPE.COMBINED
-    )
-      circle_claimed -= u.fixed_payment_amount;
-
-    const rowValues: (string | number)[] = [
-      idx + 1,
-      u.name,
-      u.address,
-      received,
-      u.sent_gifts.length
-        ? u.sent_gifts
+  return (
+    users?.map((u, idx) => {
+      const claimAmt = circleDist?.claims.find(
```

Comment:
This has an `any` type, which isn't great. Since there are so many nullish fallbacks, it might be good to type this as a number and set a fallback for it.

Reply:
updated